docs/web-component-samples/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
This is a sample using web-components and NO react. Again dummy code to just show that widgets can be used as web-components